src/Hosting/TestHost/test/ClientHandlerTests.cs

```diff
@@ -437,6 +437,28 @@ public async Task ClientCancellationAborts()
         block.SetResult();
     }
 
+    [Fact]
+    public async Task ExceptionFromDisposedRequestContent()
+    {
+        var requestCount = 0;
+        var handler = new ClientHandler(PathString.Empty, new DummyApplication(context =>
+        {
+            requestCount++;
+            return Task.CompletedTask;
+        }));
+
+        var invoker = new HttpMessageInvoker(handler);
+        Task<HttpResponseMessage> responseTask;
+        using (var message = new HttpRequestMessage(HttpMethod.Post, "https://example.com/"))
+        {
+            message.Content = new StringContent("Hello World");
+
+            responseTask = invoker.SendAsync(message, CancellationToken.None);
```

Comment:
Yes, that seemed to be happend in CI. I have adjusted the test to manually dispose the `HttpContent` but that means that it will throw in `Configure` instead of `SendRequestStream`.
I also added another test which always throws in `SendRequestStream`.